rollup.config.js

```diff
@@ -0,0 +1,55 @@
+import nodeResolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import uglify from 'rollup-plugin-uglify'
+import {sizeSnapshot} from 'rollup-plugin-size-snapshot'
+
+const input = './src/index.js'
+
+const name = 'jss'
+
+const getBabelOptions = () => ({
+  exclude: '**/node_modules/**',
+  babelrc: false,
+  presets: [['env', {modules: false}], 'stage-0', 'flow'],
+  plugins: ['external-helpers']
+})
+
+export default [
+  {
+    input,
+    output: {
+      file: 'dist/jss.js',
+      format: 'umd',
+      sourcemap: true,
+      exports: 'named',
+      name
+    },
+    plugins: [
+      nodeResolve(),
+      commonjs({include: '**/node_modules/**'}),
+      babel(getBabelOptions()),
+      replace({'process.env.NODE_ENV': JSON.stringify('development')}),
+      sizeSnapshot()
+    ]
+  },
+
+  {
+    input,
+    output: {
+      file: 'dist/jss.min.js',
+      format: 'umd',
+      exports: 'named',
+      name
+    },
+    plugins: [
```

Comment:
seems like we can reuse most of it, use production flag via cli, add uglify for production

Reply:
I write these configs for tens of project last months and this will be overreusing. It's more readable to just duplicate a couple of characters but reuse important names and babel options. Let's keep it this way for now. I'm gonna change it in next PRs.